packages/grafana-data/src/datetime/timezones.ts

```diff
@@ -341,7 +341,7 @@ const countryByCode: Record<string, string> = {
   OM: 'Oman',
   PK: 'Pakistan',
   PW: 'Palau',
-  PS: 'Palestinian Territory (Occupied)',
+  PS: 'Palestinian Territory',
```

Comment:
```suggestion
  PS: 'Palestine, State of',
```

That seems to be the most common entry for this country code. 
https://en.wikipedia.org/wiki/List_of_country_calling_codes
https://www.iban.com/country-codes

Reply:
👍